mobile/src/main/java/org/hawkular/client/android/activity/DrawerActivity.java

```diff
@@ -16,9 +16,6 @@
  */
 package org.hawkular.client.android.activity;
 
-import java.util.ArrayList;
```

Comment:
Hello @sagar15795 why was relocation of imports needed? I feel you forgot to import settings.jar in Android studio before applying reformat.

It will be very help full if you correct this by first importing the jar as instructed in contribution section.

Pre-compiled jar is at https://github.com/garg-anuj/hawkular-android-client-extras/blob/master/ide-configs/settings.jar